release-notes/unreleased/IGAPP-1147-translations.yml

```diff
@@ -0,0 +1,8 @@
+issue_key: IGAPP-1147
+show_in_stores: true
+platforms:
+  - web
+  - ios
+  - android
+en: Add finnish translations.
+de: Finnische Übersetzungen hinzugefügt.
```

Comment:
Do we need the two different release notes?

Reply:
I think its not about "needing" but more to have something in the release notes in the stores for users to see that there is actually progress :)